File: Active/Java/P010.java
// Project Euler
// Problem 10: The sum of all prime numbers below 2000000

public class P010 implements Solution<Integer> {
    public static void main(String[] args) {
        int limit = 2000_000;
        System.out.println(new P010().compute(limit));
    }

    public String compute(Integer limit) {
        Lib lib = new Lib();
        int[] array = lib.eratosthenes_sieve(limit);

        long sum = 0;
        int len = array.length;
        for (int i = 0; i < len; i++)
            sum += array[i];

        return Long.toString(sum);
    }
}

File: Active/Python/P010.py
# Find the sum of all prime numbers below 2000000

from eratosthenes_sieve import eratosthenes_sieve 

def compute(limit: int) -> str:
    primes = eratosthenes_sieve(limit) 
    sum_of_primes = sum(i for i in primes)
    return str(sum_of_primes)

if __name__ == "__main__":
    limit: int = 2000000
    print(compute(limit))
